helm-charts/falcon-sensor/templates/serviceaccount.yaml

```diff
@@ -1,12 +1,14 @@
+{{- if .Values.serviceAccount.create }}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: crowdstrike-falcon-sa
+  name: {{ include "falcon-sensor.serviceAccountName" . }}
```

Comment:
This should be `{{ .Values.serviceAccount }}`

Reply:
done